<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление заданиями</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableAssignments">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAddAssignment">
        <property name="text">
         <string>Добавить задание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEditAssignment">
        <property name="text">
         <string>Редактировать задание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnViewVariants">
        <property name="text">
         <string>Просмотр вариантов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportTest">
        <property name="text">
         <string>Экспортировать тест</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnManageStudents">
        <property name="text">
         <string>Редактировать учеников</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
